cnxepub/models.py

```diff
@@ -467,6 +486,58 @@ def references(self):
             return []
         return self._references
 
+    def fix_html_ids(self, mapping):
```

Comment:
These probably don't belong to the models ... :/
May be I need to put everything into the single html formatter?